q2_types_genomics/reference_db/_format.py

```diff
@@ -54,3 +118,15 @@ def _validate_(self, level):
 plugin.register_formats(DiamondDatabaseFileFmt, DiamondDatabaseDirFmt)
 plugin.register_semantic_type_to_format(ReferenceDB[Diamond],
                                         DiamondDatabaseDirFmt)
+
+
+class EggnogSequenceTaxaDirFmt(model.DirectoryFormat):
```

Comment:
Could we please rename this to `EggnogProteinSequencesDirFmt`? `SequenceTaxa` sounds a bit confusing imo: the fasta file is the most significant part of this artifact, with the supplementing taxa info.